packages/opencensus-core/src/stats/recorder.ts

```diff
@@ -40,6 +41,20 @@ export class Recorder {
     }
   }
 
+  /** Gets the tag values from tags and columns */
+  static getTagValues(tags: Map<TagKey, TagValue>, columns: TagKey[]):
+      TagValue[] {
+    const tagValues: TagValue[] = [];
+
+    // ignore not found key values.
```

Comment:
For tag keys that are not found, we should use `null` or `undefined` as the tag value. See https://github.com/census-instrumentation/opencensus-java/blob/5a97afdefdb2d9f4213264d91fa67b646adbf5d9/impl_core/src/main/java/io/opencensus/implcore/stats/RecordUtils.java#L86-L87.

The tag values returned from this method should have the same size as tag keys.

Reply:
This project has ```strictNullChecks``` set to false in tsconfig, hence didn't add null for missing tag values.

Reply:
The caveat here is the tag values returned must match the order of tag keys. If we skip any tag values then they became mismatched.

One example: a view is defined to break down latencies by method and caller. The tag keys are `[method, caller]` in order. If a `record` is called with only tag `caller:c1`, this method will return tag values `[c1]`. Then `c1` will be matched with `method`.